components/molecules/accordion/components/accordionItem.js

```diff
@@ -56,13 +54,22 @@ const HeaderIcon = styled(Icon)`
 `
 
 const Content = styled.div`
-  padding: 1.25rem;
+  max-height: 0;
+  overflow: hidden;
+  transition: all 500ms ease;
```

Comment:
@majid-drykiss The improvements look good, but I'm not sure about the way padding gets animated, it looks a bit weird. If we need animation, maybe animate only opacity/height?